### PR TITLE
fix: Add additional information to the Beta tags

### LINKED ISSUE #15795: Add additional information to the `Beta` tags
### Description
Add additional information to the `Beta` tags added in the application. One option could be as a _tooltip_ like in here:

[Figma](https://www.figma.com/design/bZJgpgchRfMGIKNBoyYUh8/Experimental-tag?node-id=31-3746&t=OR3ihpCDVCgfJkth-1)

The **mobile** product will incorporate this additional information like this: https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Descoped-Wallet?node-id=16352-56074&t=FaoguIBuruqiMZbn-1 and probably the **desktop** product should have the same / similar approach.

### LINKED ISSUE #15929: Beta tag is overlapping with unread indicator
# Bug Report

## Description

image

### Additional Information

- Status desktop version: 2.30 RC3
- Operating System: Windows at moment of testing

### PR DESCRIPTION
**Pull request by caybro**
### What does the PR do

- display additional beta information as a tooltip
- don't overlap the Beta badge with the unread msg indicator
- some minor cleanups & fixes

Fixes #15795
Fixes #15929

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

Badge and beta tag together:
image

Activity beta banner:
image

Settings/ENS:
image

Setting/Wallet/Account order:
image

Settings/Wallet/Manage tokens:
image

Activity center:
image

Settings/Syncing:
image

### CODE REVIEW

File: ui/imports/utils/Utils.qml [hunk 1]
```diff
@@ -137,11 +137,22 @@ QtObject {
                 `<a href="${link}">${link}</a>`
     }
 
+    function getStyledLink(linkText, linkUrl, hoveredLink, textColor = Theme.palette.directColor1, linkColor = Theme.palette.primaryColor1) {
```
Comment (alexjba) [inline ui/imports/utils/Utils.qml:140]: Any chance of unifying with `getLinkStyle`? 😄 
The 2 differences I see are the underline and link text. It's a game of spotting the difference when someone looks into these 2 functions.

**Review by alexjba — APPROVED** [review]
LGTM! Tested

**Review by igor-sirotin — APPROVED** [review]

File: ui/imports/utils/Utils.qml (hunk 1 above)
Comment (caybro) [reply ui/imports/utils/Utils.qml:140]: Yeah, I'll have a look how and where it's used :)

Comment (caybro) [reply ui/imports/utils/Utils.qml:140]: Later, we want to have it in 2.30 asap

**Pull request merged**